includes/class-amp-story-post-type.php

```diff
@@ -900,6 +900,27 @@ public static function enqueue_block_editor_scripts() {
 			);
 		}
 
+		/**
+		 * Filter list of allowed video mime types.
+		 *
+		 * This can be used by video Transcoding services to add formats they support.
+		 *
+		 * @param array Supported mime types.
+		 */
+		$allowed_video_mime_types = apply_filters( 'amp_supported_video_types', [ 'video/mp4' ] );
+
+		// If `$allowed_video_mime_types` doesn't have valid data or is empty add default supported type.
+		if ( ! is_array( $allowed_video_mime_types ) || empty( $allowed_video_mime_types ) ) {
+			$allowed_video_mime_types = [ 'video/mp4' ];
+		}
+
+		// Pass final list of allowed video types to script.
+		wp_localize_script(
+			self::AMP_STORIES_SCRIPT_HANDLE,
+			'ampAllowedVideoTypes',
```

Comment:
Let's rename this to `ampStoriesEditorSettings` and pass an array of settings instead of just the video mime types. Otherwise this doesn't scale.